cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -17,16 +17,43 @@
 import React from 'react';
 import { Theme } from 'services/ThemeHelper';
 import If from '../If';
-import { objectQuery } from 'services/helpers';
+import { isNilOrEmptyString, objectQuery } from 'services/helpers';
```

Comment:
Same as the other comment. Can you make this as a tsx file and have the styles as css-in-js? An example would be https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Password/index.tsx